src/structures/Node.ts

```diff
@@ -1256,8 +1256,10 @@ export class LavalinkNode {
     }
     /** @private util function for handling trackStart event */
     private async trackStart(player: Player, track: Track, payload: TrackStartEvent): Promise<void> {
-        player.playing = true;
-        player.paused = false;
+        if (player.get('internal_nodeChanging') !== true) { // Don't change the playing state if a nodeChange is in progress.

```

Comment:
I like to invert the if statement on falsy, change the internal_nodeChanging  value to: true|false|undefined
therefore:
```suggestion
        if (!player.get('internal_nodeChanging')) { // Don't change the playing state if a nodeChange is in progress.

```